integrations/slack/syncs/messages.ts

```
@@ -188,6 +192,24 @@ async function saveReactions(nango: NangoSync, currentChannelId: string, message
     await nango.batchSave<SlackMessageReaction>(batchReactions, 'SlackMessageReaction');
 }
 
+function removeBlockIds(data: any) {
+    // The block_id is not reliable and could change between two runs of this sync causing the messages to show as updated
+    // We remove it from the raw_json to avoid unnecessary updates.
+    // This can be reinstated if required by removing this function.
+    if (data.blocks) {
+        data.blocks.forEach((block: any) => delete block.block_id);
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)